src/app/redux/history/reducer.js

```diff
@@ -0,0 +1,51 @@
+export const initialState = {
+  history: [],
+};
+
+export function historyReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'GET_HISTORIES':
+      return state.history;
+    case 'SET_HISTORY':
+      return {
+        ...state,
+        history: [...state.history, action.payload],
+      };
+    case 'EDIT_EXPRESSION_HISTORY': {
+      let {newExpression, id} = action.payload;
```

Comment:
const

Reply:
I have serious problems with the declaration, I have already corrected it, I will be more attentive.